recipes/pyiron/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "pyiron" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "87c14d4356d9647d29adada8b31abff924f8b0caf1a213e5ce3763aba69c4393" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - pyiron_base
```

Comment:
Are these build deps?

Reply:
(I ask because it seems that most of this code is pure python, which usually doesn't have much besides python and pip as build deps).

Reply:
Oh my mistake, I thought the internal dependencies between the different packages are required but you are right it is enough to have `python` and `pip` listed there. I updated this in the latest commit https://github.com/conda-forge/staged-recipes/pull/5680/commits/b2728ead04b898b0f133abc31a59b1da30bd7de3